server/controllers/stock/inventory.js

```diff
@@ -167,6 +170,37 @@ function getInventoryItemsById(req, res, next) {
   .done();
 }
 
+/**
+* GET /inventory/metadata/receipt
+* Returns a pdf file for inventory metadata
+*
+* @function getInventoryItemReport
+*/
+function getInventoryItemReport(req, res, next) {
+  'use strict';
+
+  let request = {
+    query : req.query,
+    enterprise : req.session.enterprise,
+    project : req.session.project
+  };
+
+  core.getItemsMetadata()
+  .then((rows) => {
```

Comment:
Just as a reminder, this `.then()` call can also be written like this:
```js
.then(rows => listReceipt.buiild(rows, request))
```

It is still your choice which one you prefer.  I am just reminding you in case you didn't think of it :)